src/bci_build/package/rmt.py

```diff
@@ -31,7 +31,7 @@
         ],
         version_in_uid=False,
         support_level=SupportLevel.L3,
-        package_list=["rmt-server", "catatonit"],
+        package_list=["rmt-server", "catatonit", "bash"],
```

Comment:
Q: bash is not part of the base image already?

Reply:
It is, but this is a safety hatch, in case anyone decides to switch to a micro based build

Reply:
I believe that such change could cause a choice conflict because busybox and bash would provide /bin/sh.

Reply:
That's one of the reasons why I added it